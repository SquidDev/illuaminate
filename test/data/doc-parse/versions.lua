--- An item with some changes
-- @since 1.2
-- @changed 2.3 With some additional data
