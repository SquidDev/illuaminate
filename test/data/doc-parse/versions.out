((source versions.lua[1:1-3:41])
 (description "<p>An item with some changes</p>\n")
 (changes change (kind changed) (version 2.3) (span versions.lua[3:4-3:11])
  (description "<p>With some additional data</p>\n"))
 (changes change (kind added) (version 1.2) (span versions.lua[2:4-2:9])))
